content/docs/next-release-v2/badger.md

```diff
@@ -5,18 +5,7 @@ hasparent: true
 
 [Badger](https://github.com/dgraph-io/badger) is an embedded local storage, only available
 with **all-in-one** distribution. By default, it acts as ephemeral storage using a temporary file system.
```

Comment:
bad wording here, "all-in-one distribution", "ephemeral" (file persistence is not ephemeral)

Reply:
This was copied from existing docs, updated in latest push.